packages/meditrak-server/src/apiV2/dataElements/assertDataElementPermissions.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { hasBESAdminAccess } from '../../permissions';
+
+const getPermissionIdListWithWildcard = async (accessPolicy, models) => {
+  // Get the users permission groups as a list of ids
+  const userPermissionGroupNames = accessPolicy.getPermissionGroups();
+  const userPermissionGroups = await models.permissionGroup.find({
+    name: userPermissionGroupNames,
+  });
+  return ['*', ...userPermissionGroups.map(permission => permission.id)];
```

Comment:
What is the purpose of wildcard?

Reply:
We wanted a sort of greater scope idea of a data element that anyone can access, which is not alway necessarily the same concept as `Public`
Edwin had some examples but honestly it's been a month since I wrote this and I don't 100% remember the justification

Reply:
Got it! Thanks